docs/guides/primitive-input-output.ipynb

```diff
@@ -0,0 +1,461 @@
+{
```

Comment:
<p>It might be trivial, but I think it might be good to clarify that 1 pub -&gt; 1 result, list of pubs -&gt; list of results.</p><p> </p><p>On another note, would it make sense to have a separate definition for Sampler and Estimator PUBS? On one hand, it makes this section longer, but the current definition might make it look like the <code>Sampler</code> PUB must contain 4 values (without clarifying what to put in the place that corresponds to the observables). I also miss the introduction of <code>shots</code> as the Sampler counterpart of the Estimator <code>precision</code>.</p><p> </p><p>Alternatively to the 2 definitions, you could maybe reword "It is a tuple that contains four values" as "It is a tuple that contains up to four values depending on the selected primitive", and add the <code>shots</code> definition to the precision bullet.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/documentation/pull/1621/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>